test/echo.test.ts

```diff
@@ -70,6 +70,13 @@ Deno.test("Echo tag", async () => {
     },
   });
 
+  await test({
+    template: `
+    Hello {{-echo-}} beautiful {{-/echo-}} world!
```

Comment:
This template outputs `Hello  beautiful  world!` in current Vento; all spaces are retained.